liblangutil/UniqueErrorReporter.h

```diff
@@ -77,7 +77,8 @@ class UniqueErrorReporter
 
 	void markAsSeen(ErrorId _error, SourceLocation const& _location, std::string const& _description)
 	{
-		solAssert(!seen(_error, _location, _description), "");
+		if (_location != SourceLocation{})
+			solAssert(!seen(_error, _location, _description), "");
 		m_seenErrors[{_error, _location}] = _description;
```

Comment:
Wouldn't it be better not to mark these errors as seen at all?

Reply:
Updated